cmd/admin-trace.go

```diff
@@ -80,18 +84,20 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. Show console trace for a MinIO server with alias 'play'
-     {{.Prompt}} {{.HelpName}} -v -a play
+  1. Show verbose console trace for MinIO server
+     {{.Prompt}} {{.HelpName}} -v -a myminio
 
-  2. Show trace only for failed requests for a MinIO server with alias 'myminio'
+  2. Show trace only for failed requests for MinIO server
     {{.Prompt}} {{.HelpName}} -v -e myminio
 
-  3. Show the content of 503 failed requests
+  3. Show verbose console trace for requests with '503' status code
     {{.Prompt}} {{.HelpName}} -v --status-codes 503 myminio
 
-  4. Show all requests for a specific prefix
+  4. Show console trace for a specific path
     {{.Prompt}} {{.HelpName}} --path my-bucket/my-prefix/ myminio
 
+  5. Show console trace for requests with '404' and '503' status code
+    {{.Prompt}} {{.HelpName}} --status-codes 503 myminio
```

Comment:
The command does not comply with the example description (you need `--status-code 503 --status-code 404`)